pegasus/sites.v3/code.org/views/video_player.js.haml

```diff
@@ -1,13 +1,12 @@
-%script{src: "/js/videojs-ie8.min.js"}
-%script{src: "/js/video.min.js"}
-
 %link{href: "/css/video-js.css", rel: "stylesheet"}
 
 :javascript
 
   $(window).load(function() {
     if (window.location.search.indexOf("force_youtube_fallback") !== -1)
     {
+      $.getScript("/js/videojs-ie8.min.js");
+      $.getScript("/js/video.min.js");
```

Comment:
I tested programmatically loading the scripts 2 different ways, I used jQuery and the document.createElement:

   $.getScript("/js/videojs-ie8.min.js");
    $.getScript("/js/video.min.js");

and 
         var doc = document,
          video = doc.createElement("script"),
          video_ie = doc.createElement("script");
          video.type = "text/javascript";
          video.src = "/js/video.min.js";
          doc.body.appendChild(video);
          video_ie.type = "text/javascript";
          video_ie.src = "/js/videojs-ie8.min.js";
          doc.body.appendChild(video_ie);

I executed the test you suggested to click on "Watch the Video" before the video.min.js file was loaded and they behaved the same.

If a user clicks on the "Watch the Video" button before the video.min.js file is added to the DOM :
- the video player renders but does not start
- the user has to click the play button

The current behavior is, a user clicks on the "Watch the Video" button and the video player renders and starts up. 

I think the change in behavior is related to the video.min.js being loaded later and "Watch the Video" button is enabled.  Maybe I should look into the button only being enabled when the script has been loaded if this behavior is not acceptable. 

I have attached screenshots of where the files videojs-ie8.min.js and video.min.js are loaded from code.org and my machine with the changes. 
![aftercodechanges](https://cloud.githubusercontent.com/assets/28658347/26211944/bb45936c-3ba8-11e7-9e6d-60403e5bed17.PNG)
![codeorginproduction](https://cloud.githubusercontent.com/assets/28658347/26211954/c7526644-3ba8-11e7-8fa1-f93453792fc6.PNG)

There are two main differences with the two ways I implemented the change:
With jQuery:
-	Asynchronous
-	Status code is still 200
With document.createElement:
-	Synchronous
-	Status code is 304, I have not found a good explanation as to why

Please let me know your preference of how the code should be implemented and if I should look into only enabling the button if the script has been loaded.